crates/fiber-lib/src/fiber/config.rs

```diff
@@ -507,6 +509,13 @@ impl FiberConfig {
         self.sync_network_graph
             .unwrap_or(DEFAULT_SYNC_NETWORK_GRAPH)
     }
+
+    pub fn gen_node_features(&self) -> FeatureVector {
```

Comment:
What's the usage of this function?

Reply:
https://github.com/nervosnetwork/fiber/blob/9b9c02c6de31555e7934327415a96406573a9e3a/crates/fiber-lib/src/fiber/network.rs#L3615

here used to initialize features.

Reply:
Why two sets of "default" feature vectors?

Reply:
here I left a place for configuration file to override default feature vector.